druid-shell/src/gtk/mod.rs

```diff
@@ -45,14 +46,31 @@ pub mod menu;
 pub mod util;
 pub mod win_main;
 
+macro_rules! clone {
```

Comment:
Can we add a comment to this, explaining what it's for? will be a point of confusion for anyone reading this code in the future.